src/main/res/xml/settings_profile.xml

```diff
@@ -12,6 +12,12 @@
         android:key="@string/settings_profile_nickname_key"
         android:title="@string/settings_profile_nickname"
         app:useSimpleSummaryProvider="true" />
+    
+    <EditTextPreference
```

Comment:
The requirement:
Height: (feet or metric depending Settings->Default Units…)

My feedback:
Should add label indicating the unit (metre/m or foot/ft) in the dialog/text field. Use getUnitSystem() in PreferencesUtils to get the current setting.